pipeline_steps/tempest.groovy

```diff
@@ -1,21 +1,20 @@
 def tempest_install(vm=null){
   // NOTE(mkam): Can remove ANSIBLE_CACHE_PLUGIN when we no longer gate stable/mitaka
+  // NOTE(alextricity25): --skip-tags needs to be used here because tags cannot be used
+  // with included roles in earlier versions of Ansible. Such that used in Mitaka
   common.openstack_ansible(
     vm: vm,
-    playbook: "os-tempest-install.yml",
-    path: "/opt/rpc-openstack/openstack-ansible/playbooks",
+    playbook: "../../scripts/run_tempest.yml",
+    args: "--skip-tags tempest_execute_tests",
+    path: "/opt/rpc-openstack/rpcd/playbooks",
     environment_vars: ["ANSIBLE_CACHE_PLUGIN=memory"]
   )
 }
 
 def tempest_run(wrapper="") {
   def output = sh (script: """#!/bin/bash
-  utility_container="\$(${wrapper} lxc-ls |grep -m1 utility)"
-    ${wrapper} lxc-attach \
-      --keep-env \
-      -n \$utility_container \
-      -- /opt/openstack_tempest_gate.sh \
-      ${env.TEMPEST_TEST_SETS}
+  ${wrapper} cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible \
```

Comment:
So the tricky part about this wrapper business is that when there is a wrapper, this becomes:
`sudo ssh -T -oStrictHostKeyChecking=no deploy1 cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible`

Which executes just the cd on the deploy1 VM and openstack-ansible on the host node, instead of both commands on the VM. If you put single quotes around both commands, though, I'm pretty sure that'll work for both when there's a wrapper and when there isn't.